keymage.js

```diff
@@ -7,9 +7,10 @@
 
 (function(define, undefined) {
 define(function() {
+    'use strict'
+
     var VERSION = '1.1.3';
-    var isOsx = typeof navigator !== 'undefined' &&
-        ~navigator.userAgent.indexOf('Mac OS X');
+    var isOsx = ~navigator.userAgent.indexOf('Mac OS X');
```

Comment:
I'm really wondering why did I check for navigator to be present... I guess so that you could import that in node. Not sure that has any relevance.